### ISSUE TITLE
Information about the learning rate

### ISSUE THREAD

**Issue by SebastianJanampa** [roboflow/rf-detr#255]
Hello,

Thank you for your work. I was reading the training hyperparameters, and i noticed that the learning rate for DINOv2 is greater than the learning rate for the decoder. In previous DETR models, the backbone usually has a smaller learning rate than the other parameters. Why did you choose a learning rate of 0.2 for dinov2.

https://github.com/roboflow/rf-detr/blob/ed54776d02698b8bace3a7cfe998029ed8fffa2f/rfdetr/config.py#L55-L57
